components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/SelfRegistrationConfigImpl.java

```diff
@@ -30,6 +32,12 @@
 import java.util.Map;
 import java.util.Properties;
 
+import static org.wso2.carbon.identity.governance.IdentityGovernanceUtil.getPropertyObject;
+
+/**
+ * Class which contains self sign up configs.
+ */
+
```

Comment:
Shall we remove the extra newline?

Reply:
Fixed in commit ec6255cd047f1d4cb8fd4ed1c247829f4b1fd46d